AppCenter/AppCenterTests/MSDBStorageTests.m

```diff
@@ -92,7 +92,6 @@ - (void)testInitWithSchema {
 
 - (void)testTableExists {
   [self.sut executeQueryUsingBlock:^int(void *db) {
-
```

Comment:
Why are all the blank lines before comments removed?  Anyways you should probably just revert this file since there are no functional changes

Reply:
That may have been the formatter. I propose that we leave it because if we don't, this will come up every time